metadata/provider_metadata.json

```diff
@@ -61265,10 +61265,10 @@
             {
```

Comment:
Don't update this file we need update during release its auto generated

Reply:
ok